### PR TITLE
Just a bit of cleanup/modernization

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/recipes-i/round-numbers-to-n-decimals.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/recipes-i/round-numbers-to-n-decimals.md?ref=3936f009aadb63a256d7b8627c24f9f90324c1c5)

File: javascript/core/recipes-i/round-numbers-to-n-decimals.md [hunk 1]
```diff
@@ -1,28 +1,15 @@
...
-
 aspects:
   - introduction
   - workout
 
 
-
 ---
 
 # Round numbers to `N` decimals
```
Comment (kapnobatai137) [inline javascript/core/recipes-i/round-numbers-to-n-decimals.md]: should we capitalize headlines? (I've been doing this for insights that I update/fix)

```suggestion
# Round Numbers to `N` Decimals
```

Comment (nem035) [reply javascript/core/recipes-i/round-numbers-to-n-decimals.md]: I'm fine either way, you decide

**Comment by kapnobatai137** [comment]
will improve some more on the content, leaving this for later

File: javascript/core/recipes-i/round-numbers-to-n-decimals.md [hunk 2]
```diff
@@ -1,31 +1,18 @@
...
 aspects:
   - introduction
   - workout
 
 
-
 ---
 
-# Round numbers to `N` decimals 
+# Round Numbers to `N` Decimals
```
Comment (Loopiezlol) [inline javascript/core/recipes-i/round-numbers-to-n-decimals.md:15]: is this a new headline convention? we capitalize every non-filler word?

Comment (kapnobatai137) [reply javascript/core/recipes-i/round-numbers-to-n-decimals.md:15]: I've been trying to put all headlines in title case, using the apa guidelines. I think this looks better, but also it is just a preference

**Comment by nem035** [comment]
Good to merge? @kapnobatai137

**Pull request merged**